etc/README.md

```diff
@@ -0,0 +1,22 @@
+# Installation instructions
+
+## zsh
+
+Create a new folder for completions:
+
+```sh
+mkdir -p ~/.zsh/completions
+```
+
+Download the file hub.zsh_completion and rename it to `_hub`:
+
+```sh
+curl https://github.com/github/hub/blob/master/etc/hub.zsh_completion >  ~/.zsh/completions/_hub
```

Comment:
This needs to follow redirects by adding the `-L` flag. But why download it from the server when people should probably already have hub downloaded on their computer? Maybe it would be better to instruct them to simply copy the `etc/hub.zsh_completion` from their download the the zsh completions location.